src/components/Search/SearchDropDown.tsx

```diff
@@ -0,0 +1,78 @@
+import { FunctionComponent, useState } from 'react'
```

Comment:
`useState` is not used in the component 😜
```suggestion
import { FunctionComponent } from 'react'
```